index.html

```diff
@@ -0,0 +1,269 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Asad Shareef - Portfolio</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+
+  <header>
+    <nav class='navigation'>
+      <a href="index.html" class="title-top">Welcome</a>
+      <div class="nav-button">
+        <img src="assets/Union.png" />
+      </div>
+    </nav>
+  </header>
+
+
+  <section class="content">
+    <section class="title-body">
+      <h1>Hey there. I'm Asad</h1>
+      <h2 class='highlited-title'>I'm a software developer</h2>
+    </section>
+
+    <p class='about-me-brief'>
+      I can help you build a product , feature or website Look through some of my work and experience! If you like
+      what
+      you see and have a project you need coded,
+      don’t hestiate to contact me.
+    </p>
+
+    <section class='social-icons'>
+      <ul>
+        <li>
+          <a target="_blank" href="#"><img src="assets/twitter.png"
+              alt="Twitter" /></a>
+        </li>
+        <li>
+          <a target="_blank" href="#"><img src="assets/Linkedin icon.png"
+              alt="LinkedIn" /></a>
+        </li>
+        <li>
```

Comment:
- [ ] Kindly make sure you use the semantics tag for accessibility on your page. You may consider grabbing all the introduction part in a `section` tag and all the projects in one `section`. 
You do not need to grab all small parts in a section.
`Hint: You may consider checking this` [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) `to know more about where you can use the section tag.`

Reply:
@christianbiring1
I update the section to div in line number 23 to +46.

Reply:
@christianbiring1 
I updated the section tag from +23 to +46.